package.json

```diff
@@ -1,20 +1,39 @@
 {
-  "private": true,
-  "scripts": {
-    "prepare": "yarn tslint && lerna run prepare",
```

Comment:
could you include back the tslint and tslint-fix script ?

Reply:
done

Reply:
could you include yarn tslint by default in prepare ?